src/sentry/spans/consumers/process/factory.py

```diff
@@ -71,20 +72,28 @@ def _build_snuba_span(relay_span: Mapping[str, Any]) -> MutableMapping[str, Any]
     )
 
     sentry_tags: MutableMapping[str, Any] = {}
+
     if tags := relay_span.get("data"):
         for relay_tag, snuba_tag in TAG_MAPPING.items():
             if relay_tag in tags:
                 sentry_tags[snuba_tag] = tags.get(relay_tag)
+
+    if "op" not in sentry_tags:
+        sentry_tags["op"] = relay_span["op"]
```

Comment:
If we want to be consistent with the previous `spans_processor` in Snuba then I think this is correct, see https://github.com/getsentry/snuba/blob/f429b0fe1a10f6c7c9c046885a292710b4628217/snuba/datasets/processors/spans_processor.py#L262.

I would make the operation infallible though, because there is no formal guarantee that this key is set on the relay span (same for `"status"`).

```suggestion
    if "op" not in sentry_tags:
        sentry_tags["op"] = relay_span.get("op", "")
```